### PR TITLE
[test] Tag checks that use external resources

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix https://jira.cscs.ch/browse/UES-572

### CODE REVIEW

**Comment by jgphpc** [comment]
To be clear, i will also add the `craype` tag but in another PR.

**Comment by vkarak** [comment]
@jgphpc Rename the tag to `external-resources` and then we can merge it.

File: cscs-checks/libraries/math/scalapack_compile_run.py [hunk 1]
```diff
@@ -68,31 +68,31 @@ def scalapack_sanity(number1, number2, expected_value):
...
+#                 sn.extractall(r'GFLOPS/s:\s+(?P<gflops>\S+)',
+#                               self.stdout, 'gflops', float)
+#             )
+#         }
+#
+#         self.reference = {
+#             'monch:compute': {
+#                 'perf': (24., -0.1, None)
+#             }
+#         }
```
Comment (vkarak) [inline cscs-checks/libraries/math/scalapack_compile_run.py]: Remove it completely. Git will keep the history.

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Pull request merged**